doc/UsersGuide/source/InputsOutputs.rst

```diff
@@ -1440,6 +1707,50 @@ A brief example of the diag_table is shown below.  ``“...”`` denotes where l
    "ocean_model", "tauy",      "tauy",          "ocn%4yr%2mo%2dy%2hr","all",.true.,"none",2
    ...
 
+-------
+HYCOM
+-------
+
+HYCOM output configuration is set in the :ref:`blkdat.input<HYCOM_BlkdatInput>` file. A few common configuration options are described in :numref:`Table %s <HYCOM_OutputConfig>`
+
+.. _HYCOM_OutputConfig:
+
+.. list-table:: *The following table describes HYCOM output configuration.*
+   :widths: 10 25
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - dsurfq
+     - Configure the output frequency for archs files
```

Comment:
Can we put the same information that you get if you follow the link to the HYCOM documentation:

dsurfq Number of days between model diagnostics at the surface.
diagfq Number of days between model diagnostics.
rstrfq Number of days between model restart output

I couldn't find the variable ``meanfq``.